esx_service/utils/auth_api.py

```diff
@@ -190,7 +195,8 @@ def modify_privileges(privileges, allow_create, volume_maxsize_in_MB, volume_tot
     logging.debug("modify_privileges: allow_create=%s, volume_maxsize_in_MB=%s, volume_totalsize_in_MB=%s",
                   allow_create, volume_maxsize_in_MB, volume_totalsize_in_MB)
     if allow_create is not None:
```

Comment:
Isn't None a valid value (don't change the privilege)?
boolean, string case insensitive true/false values mean to change the privilege. Rest everything is invalid.